TileSystem/Interfaces/Base/IEntity.cs

```diff
@@ -19,6 +21,7 @@ public interface IEntity
 		string Variation { get; }
 
 		ITile Tile { get; }
+		void SetParent(ITile tile);
```

Comment:
SetParent ... not sure about the naming ...
Hard to see the tile as a parent of a entity :P

SetPosition / SetTile / ...